tap_hubspot/streams.py

```diff
@@ -127,7 +133,34 @@ class PropertiesContactsStream(PropertiesStream):
     path = "/crm/v3/properties/contacts"
 
 
+class AssociationsDealsToCompaniesStream(HubspotStream):
+    name="associations_deals_companies"
+    path = "/crm/v4/objects/deals/{deal_id}/associations/companies"
```

Comment:
Are we at v4 already? 🤣

Reply:
Yeah, things are canging quite quickly :p